README.md

```diff
@@ -1622,6 +1666,13 @@ const client = createClient({
 
 </details>
 
+## Web IDE support
+
+`graphql-ws` is supported by:
+
+- [Apollo Studio Explorer](https://www.apollographql.com/docs/studio/explorer/additional-features/#subscription-support); select `graphql-ws` in Connection Settings
+- [GraphiQL](https://github.com/graphql/graphiql); use [`createGraphiQLFetcher`](https://github.com/graphql/graphiql/blob/main/packages/graphiql-toolkit/docs/create-fetcher.md)
+
```

Comment:
Is a new section really necessary?

Both of these can be recipes:
- Client usage with Apollo Studio Explorer
- Client usage with GraphiQL

Reply:
Ah sounds good, I'll change it tomorrow. I thought you wanted to keep the recipes as source code only :)

Reply:
I am keen on keeping the README compact. The two can also be seen as recipes:
- Client usage with Apollo Studio Explorer
  - Select "graphql-ws" in Connection Settings (you can even attach a screenshot 🤷‍♂️)
- Client usage with GraphiQL
  - Source code of how to actually use the `createGraphiQLFetcher` with WS. Can be mostly copied from [their "Adding graphql-ws websockets subscriptions" section](https://github.com/graphql/graphiql/blob/main/packages/graphiql-toolkit/docs/create-fetcher.md#adding-graphql-ws-websockets-subscriptions). However, one thing I'd change is leverage the `wsClient` property to help people understand how they can configure `graphql-ws` for GraphiQL.

Reply:
Great, updated. I didn't add a screenshot because I wasn't sure of an appropriate place to put the image (a file in the repo?). There's an image in the linked docs at least though.